packages/tooling/src/oas.js

```diff
@@ -67,7 +67,7 @@ class Operation {
       }, {});
   }
 
-  getHeaders() {
+  getHeaders(includeContent = false) {
```

Comment:
@erunion only other thing I'm debating is how we should wrap this logic (if at all) — currently I just wrapped it in a flag. Let me know if you have any better ideas.

Reply:
I've been trying to think of a scenario where we wouldn't want to return the accept/content-type, and I'm not sure. Maybe if you just wanted the explicitly-defined headers? But then again, request and response content-types are defined explicitly.

Reply:
Yeah I wasn't sure how we draw the line here... since this feels more like an inference than an explicitly-defined header

Reply:
In metrics, if we're always going to do `includeContent=true` then we might as well just always return it.

Reply:
Cool, I'll revert the flag stuff and just make it so it always returns these.